### DISCUSSION TITLE
Using a domain purchased through Vercel

### DISCUSSION THREAD

**Discussion by jessicagallagher** [vercel/community#4538] [Help] [answered]
### Summary

I'm trying to connect my domain that I purchased to my Medium account. I have a paid Medium account, and clicking on the option to use a custom domain name gives me two DNS records, which I've successfully added (and proofread) to the domain on Vercel. This is probably the 3rd or 4th time I've tried this. When I go to Medium to check to see if it's been linked, I get a message saying that my domain couldn't be linked. When I visit the domain itself, I get an error stating: ERR_SSL_VERSION_OR_CIPHER_MISMATCH and also: The client and server don't support a common SSL protocol version or cipher suite. If I add www in front of my domain, I get a 404 error saying that the deployment couldn't be found.

I checked Medium's troubleshooting docs, and nothing there relates to my error message. They also provide examples around GoDaddy, Namecheap, and Google Domains—so I'm not sure if purchasing my domain through Vercel is even possible. When I search for answers or suggestions regarding this, all I get is Medium articles about purchasing domains through Vercel.

Has anyone else encountered this issue? Any suggestions?

### Steps to Reproduce

- Visit medium.com and click on settings
- Click on connect your own domain
- Add DNS records to Vercel
- Check medium to see if it was a success after 3+ days
- Receive error messages mentioned in the above summary

**Comment by amyegan** [comment]
Hi @jessicagallagher. That error usually means there is a missing or invalid certificate. The tricky part is there are a lot of possible causes so I can't be sure what's causing it without a bit more info. If you share the domain, I can look into it for you.

**Reply by jessicagallagher** [reply]
Sure thanks for looking into this for me. The domain is letsgetthismerged.com

**Reply by amyegan** [reply, marked as answer]
It looks like there are some DNS records still pointing at Vercel. That's probably causing the conflict. If you remove any records that point to `76.76.21.21` or `cname.vercel-dns.com.`, does that work for you?

**Reply by jessicagallagher** [reply]
That worked. Thanks for your help.
